docs/use-cases/system-use-cases.md

```diff
@@ -0,0 +1,31 @@
+The use cases documented on this page focus on how the system is supposed to work from the perspective of the end user.
+
+### Use case: Enroll contactless card to receive transit benefit
+
+**Primary Actor**: Transit rider
+
+**Systems**: Benefits app, payment processor
+
+**Preconditions**:
+
+- Transit rider has confirmed their eligibility with the Benefits app
+- Transit rider has their contactless card information available
+- Benefits app is able to contact the payment processor
+
+**Trigger**: Transit rider initiates the enrollment process
+
+**Basic flow**:
+
+1. Transit rider enters their contactless card information
+2. Benefits app passes that information to the payment processor to enroll the card
+3. Payment processor successfully enrolls card
+
+**Alternate flows**:
+
+- 3a. Payment processor returns with one of the following errors: address verification failed, token is invalid, or general server error
+    - 3a1. Transit rider chooses to retry, starting back at initiating the enrollment process
```

Comment:
`3a1` means that the user has to have a Button to retry right? Or be allowed to use the Back button Browser?

Reply:
Yeah, `3a1` means that the app needs to provide the user with some way to go back to initiate the enrollment process. In the current implementation, this is manifested as a button that takes the user back to `/enrollment`.

As for the question about the browser back button, I would guess that since our app has a "Previous page" button on some pages, this implies that our UX philosophy is to not solely rely on the browser's back button.